packages/meso-js/README.md

```diff
@@ -317,8 +318,9 @@ type TransferConfiguration = {
   network: Network; // The network to use for the transfer
   walletAddress: string; // The user's wallet address obtained at runtime by your application
   layout?: Layout; // Configuration to customize how the Meso experience is launched and presented
-  onSignMessageRequest: (message: string) => Promise<SignedMessageResult>; // A callback that is fired when you need to collect the user's signature via their wallet.
-  onEvent?: (event: MesoEvent) => void; // An optional handler to notify you when an event or error occurs. This is useful for tracking the state of the user through the experience.
+  headlessSignature?: boolean; // Automatically invoke the `onSignMessageRequest` callback to initiate message signing
```

Comment:
I wonder if we could say something here like:

```suggestion
  headlessSignature?: boolean; // Perform message signing in the background without prompting the user. This is useful for embedded wallets.
```

I'd like to convey the _why_ of this in documentation as well as the _what_